### PR TITLE
Sqlizer

### PR DESCRIPTION
**Pull request by PatriciaGN**

### CODE REVIEW

File: api/package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
-  "name": "api",
+  "name": "Votewatch",
```
Comment (eggs-benny) [inline api/package.json:2]: Need to update to Aye Spy

File: api/package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
   "version": "1.0.0",
-  "description": "",
+  "description": "Votewatch backend server",
```
Comment (eggs-benny) [inline api/package.json:4]: Here too

File: api/package.json [hunk 3]
```diff
@@ -13,13 +13,17 @@
   "dependencies": {
     "@react-navigation/native": "^6.0.14",
     "@react-navigation/native-stack": "^6.9.2",
+    "bcryptjs": "^2.4.3",
     "body-parser": "^1.20.1",
```
Comment (eggs-benny) [inline api/package.json:17]: this can be removed as no longer used

File: frontend/src/components/context/AuthContext.js [hunk 4]
```diff
@@ -1,43 +1,73 @@
...
-  //   } catch (error) {
-  //     console.log(`Logged in error ${error}`);
-  //   }
-  // };
-
-  // useEffect(() => {
-  //   isLoggedIn();
-  // }, []);
+  const isLoggedIn = async () => {
+    console.log(
```
Comment (eggs-benny) [inline frontend/src/components/context/AuthContext.js:51]: do we still need this console log?

**Review by eggs-benny — APPROVED** [review]

**Review by JoeOsborne77 — APPROVED** [review]

**Pull request merged**